apps/common/components/Apps.tsx

```diff
@@ -0,0 +1,77 @@
+import Image from 'next/image';
+import vaultsManifest from 'public/apps/vaults-manifest.json';
+import ybribeManifest from 'public/apps/ybribe-manifest.json';
+import ycrvManifest from 'public/apps/ycrv-manifest.json';
+import homeManifest from 'public/manifest.json';
+import {VAULTS_MENU} from '@vaults/constants/menu';
+import {YCRV_TOKEN_ADDRESS} from '@yearn-finance/web-lib/utils/constants';
+import LogoYearn from '@common/icons/LogoYearn';
+import {YBRIBE_MENU} from '@yBribe/constants/menu';
+import {YCRV_MENU} from '@yCRV/constants/menu';
+
+import type {ReactElement} from 'react';
+import type {TMenu} from '@yearn-finance/web-lib/layouts/Header.next';
+import type {TMetaFile} from './Meta';
+
+export enum AppName {
+	VAULTS = 'vaults',
+	YCRV = 'yCRV',
+	VEYFI = 'veYFI',
+	YBRIBE = 'yBribe'
+}
+
+type TApp = {
+	name: AppName;
+	href: string;
+	menu: TMenu[];
+	manifest: TMetaFile;
+	icon: ReactElement;
+}
+
+export const APPS: { [key in AppName]: TApp } = {
```

Comment:
Noticed that we already had here a lot of static data, so made this a first class citizen as we have everything in a single place.

It allowed to DRY the code quite a bit